crates/builder-api/src/lib.rs

```diff
@@ -180,6 +180,10 @@ pub fn with_endpoints(router: Router<State>) -> Router<State> {
             latest_solution_failures::PATH,
```

Comment:
Maybe we can change this old fn to `latest_failures_for_solution` or similar in a future PR to more clearly distinguish the two failure queries